src/ralph/models/xapi/video/fields/results.py

```diff
@@ -53,61 +54,73 @@ class VideoSeekedResultExtensionsField(BaseModelWithConfig):
         timeTo (float): Consists of the point in time the actor changed to in a media
             object during a seek operation.
         length (float): Consists of the actual length of the media in seconds.
-        playedSegments (str): Consists of parts of the video the actor watched during
-            current registration in chronological order.
         progress (float): Consists of the percentage of media consumed by the actor.
     """
 
-    timeFrom: Optional[float] = Field(alias=VIDEO_EXTENSION_TIME_FROM)
-    timeTo: Optional[float] = Field(alias=VIDEO_EXTENSION_TIME_TO)
+    timeFrom: float = Field(alias=VIDEO_EXTENSION_TIME_FROM, default_factory=float)
+    timeTo: float = Field(alias=VIDEO_EXTENSION_TIME_TO, default_factory=float)
     length: Optional[float] = Field(alias=VIDEO_EXTENSION_LENGTH)
-    playedSegments: Optional[str] = Field(alias=VIDEO_EXTENSION_PLAYED_SEGMENTS)
```

Comment:
playedSegments is not in the template description, and length neither by the way ! 
https://profiles.adlnet.gov/profile/90b2c849-d744-4d0c-8bd0-403e7859a35b/templates/d0ee8f46-5704-428c-b21e-b583f99cd571

Reply:
You are right! It seems there are differences between the video profile on the ADL profile server and the [video profile documentation (recipe)](https://liveaspankaj.gitbooks.io/xapi-video-profile/content/statement_data_model.html).

This might be because the video profile on the ADL profile server is version `https://w3id.org/xapi/video/v1.0.2`
and the documentation for the video profile is for version `https://w3id.org/xapi/video/v1.0.3` :confused:

Also, looking at the [ADL profile server user guide](https://adlnet.gov/guides/xapi-profile-server/user-guide/Profiles.html#profile-maintenance), it seems that once a profile is published, it cannot be updated (a part of adding translations).
Thus, could it be that the video profile might not be updated to `v1.0.3` on the ADL profile server?

Could we add a note in the xAPI video statements model that they are using version `1.0.2` of the video profile?

Reply:
Where do you see in the documentation the v1.0.3 version ? I don't get it. Only the 1.0.2 version has been published on the ADL server, I don't get why they would have documented a newer version before publishing it :disappointed:

Reply:
It seems they indeed don't mention a version number on their gitbook, however, the [recipe for the video seeked statement](https://github.com/liveaspankaj/xapi-video-cop/blob/master/statement_data_model.md?plain=1#L600-L624) looks very similar to [the `v1.0.3` jsonld template](https://github.com/adlnet/xapi-authored-profiles/blob/master/video/v1.0.3/video.jsonld#L462-L504) of the video profile on the [xapi-authored-profiles](https://github.com/adlnet/xapi-authored-profiles) repository.
I don't know why they published `v1.0.2` on the profile server instead of `v1.0.3`. Maybe it's more widely accepted/used?

Reply:
It's very unclear indeed. I based the model on v1.0.2 as it is the more visible on the profile documentation. But indeed, it would change a bit if the v1.0.3 is the official new version for this profile. I can ask in a issue the reason why. WDYT?

Reply:
Good Idea) I think version v1.0.2 should be fine, however, it's always good to double check)

Reply:
@SergioSim I opened [this issue](https://github.com/adlnet/xapi-authored-profiles/issues/93) in the xapi-authored-profile repository.